Controllers/InputModels/NoteInputModel.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Taskmanager.Data.Entities;
+
+namespace Taskmanager.Controllers.InputModels
+{
+    public class NoteInputModel 
+    {
+        [Required]
+        [StringLength(300, MinimumLength = 1, ErrorMessage = "Note must be between 1 and 300 characters")]
+        public string Body { get; set; }
+    
+        public static Note MapNote(NoteInputModel inputModel)
+        {
+            if(inputModel != null)
+            {
+                return new Note{ Body = inputModel.Body };
+            }
+
+            throw new Exception("Note was empty");
```

Comment:
It's bad practice to throw default Exception class. Exception system has a lot of different subclasses like NullReferenceException, etc, and you should choose depending on the situation